extensions/amp-consent/integrating-consent.md

```diff
@@ -144,6 +146,18 @@ One can get access to the client information via the name attribute inside the i
 info = JSON.parse(window.name);
 ```
 
+#### consentMetadata
+
+`consentMetadata` allows for additional consent information to be passed to `<amp-consent>` to be give to ad vendors and other extensions. `consentMetadata` can be received from the [external iframe consent flow](#informingconsentresponse) outlined above or from the `checkConsentHref` endpoint response. `consentMetadata` will only be updated if a `consentString` is passed in as well. `consentMetadata` will also be sent along with the information in [`onUpdateHref`](./amp-consent#onUpdateHref). `consentMetadata` will be stored in localStorage and will be affected one-off tradeoff described in [client caching](./amp-consent#Client-caching). The only fields that are valid are as follows:
```

Comment:
What about 
`<amp-consent>` [caches](./amp-consent#Client-caching) and [passes](link to the  additional getConsentMetadata  doc)consent information to vendors via `consentMetaData` object along with a non-empty `consentString`.  You can find an example of the `consentMetaData` and its supported fields below.

Reply:
Much more concise! Thanks